Packs/CommonScripts/ReleaseNotes/1_14_45.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### SendEmailOnSLABreach
+
+- Removed support for xsiam marketplace.
```

Comment:
this line won't be seen in XSIAM (since SendEmailOnSLABreach isn't XSIAM content anymore) so I wouldn't even mention it here